### DISCUSSION TITLE
Lora not loading

### DISCUSSION THREAD

**Discussion by Kelvinlu95** [lllyasviel/Fooocus#1108] [Q&A] [answered]
Hi, I downloaded some Lora from civitai and put it in the Lora folder but from the console it never load the Lora but only requested it.

**Comment by botzero-net** [comment, marked as answer]
Make sure it is a SDXL LoRA and that you're using its trigger word.  Example:
https://civitai.com/models/133573/taco-dog
![civit](https://github.com/lllyasviel/Fooocus/assets/61260167/fa4c5329-420c-45d6-be14-d7418f3ea120)

**Comment by Kelvinlu95** [comment]
Thanks!
